src/gui/editorwidgets/qgstexteditwrapper.cpp

```diff
@@ -74,6 +77,20 @@ QVariant QgsTextEditWrapper::value() const
     // input rather then discarding it entirely
     return QVariant( v.left( field().length() ) );
   }
+  else if ( field().type() == QVariant::Map )
+  {
+    json j = json::parse( v.toUtf8(), nullptr, false );
```

Comment:
I think that will trigger an unrequired extra conversion

```suggestion
    json j = json::parse( v, nullptr, false );
```

Reply:
Hmm, not sure about that. I get a `static assertion failed: each element in the iterator range must have the size of 1 byte` error without the toUtf8() call

Reply:
Sorry, I was thinking it's `QgsJsonUtils::parseJson`, better use that one (or `v.toStdString()`)
But maybe it's easier to use `json::accept` anyway to just check if it's valid json?